### PR TITLE
scripts: pjsip: add a migration script for reg.conf

### PR DESCRIPTION
**Pull request by pc-m**
given a reg.conf that overrides registration sections this script will try to
find matching endpoints and update there configuration

### CODE REVIEW

File: scripts/import_pjsip_config.py [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+argument_parser = argparse.ArgumentParser(
+    'Import an Asterisk pjsip configuration file',
+)
+argument_parser.add_argument('filename', help='The file to import')
+
+
+def migrate_section(confd_client, block_name, options):
+    # if the names is reg_<endpoint_name>@host we are in a registration section
+    if 'reg_' and '@' in block_name:
```
Comment (fblackburn1) [inline scripts/import_pjsip_config.py:48]: `'reg_'` is always evaluate to `True` :thinking: 
```suggestion
    if block_name and block_name.startswith('reg_') and '@' in block_name:
```

File: scripts/import_pjsip_config.py [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+        endpoint_name = None
+        section = 'registration_section_options'
+    else:
+        print('ignoring', block_name, 'invalid section, only registration sections can be overridden')
+        return
```
Comment (fblackburn1) [inline scripts/import_pjsip_config.py:53]: Alternative
```suggestion
    if not block_name or not block_name.startswith('reg_') or  '@' not in block_name:
        print('ignoring', block_name, 'invalid section, only registration sections can be overridden')
        return
    
    endpoint_name = None
    section = 'registration_section_options'
```

**Review by fblackburn1 — CHANGES_REQUESTED** [review]
We should try to keep the same standard from wazo-upgrade scripts with `-` vs `_` for the script name
https://github.com/wazo-platform/wazo-upgrade/#upgrade-scripts

**Comment by pc-m** [comment]
Fixed, thanks!

**Review by fblackburn1 — APPROVED** [review]

**Pull request merged**